src/scss/cs/isaac.scss

```diff
@@ -508,11 +508,171 @@ a.browse-events{
   align-items: center;
   background-color: white; 
   border: 3px solid #D60265; 
-  padding: 10px 15px;
+  padding: 7px 48px;  
   border-radius: 5px;
 }
 
+// ----------------------------------------------
```

Comment:
A couple of things here: 

1. This could be in the `homepage.scss` file to prevent bloating this main isaac file out more. Or if you're comfortable with doing so, you could create a new file that gets imported to match the new component naming if you go ahead with that.
2. Linked to the above, all of these classes could be nested within the main wrapper. This would have two benefits: 
a) all of the resources block specific styling is scoped properly and will prevent any unintended styling clashes. 
b) you'll be making use of the scss pre-processing better, and will be to take advantage of the nesting system and the ability to use the parent selector (`&`) reference. 

Shout if you need more explanation for that :)

Reply:
I have added all the styling to the `homepgae.scss` file. I will keep this in mind working on other part of the homepage too. 
I have nested all the classes within the main `.resource-section` class.